libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -591,6 +637,31 @@ bool AC_PrecLand::retrieve_los_meas(Vector3f& target_vec_unit_body)
             // Apply sensor yaw alignment rotation
             target_vec_unit_body.rotate_xy(radians(_yaw_align*0.01f));
         }
+
+        // camera frame to body frame conversion
+        switch (_orient.get()) {
+            case ROTATION_NONE:
+                target_vec_unit_body.rotate(ROTATION_PITCH_90);
+                break;
+            case ROTATION_YAW_180:
+                target_vec_unit_body.rotate(ROTATION_PITCH_270);
+                target_vec_unit_body.rotate(ROTATION_ROLL_180);
```

Comment:
Why not `ROTATION_YAW_180`?!

Reply:
If we don't do any rotation, precland construct target vector such that camera is mounted downwards. So, let's say if the camera is mounted forward, we need to rotate the vector constructed downwards to front by rotating it by PITCH_90 rotation. 
Now, for the case the camera is mounted on the back we will get correct vector by the combination of these two rotations I did above. But yes, I agree that YAW_180 rotation is more intuitive. So, we can do it by rotating it by PITCH_90 (down to front) and then YAW_180 (front to back). This would also give same result.